src/explore-education-statistics-admin/src/pages/release/edit-release/manage-datablocks/tableUtil.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable */
```

Comment:
Can we remove the full file lint ignore?

Reply:
yes

Reply:
removed